src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static birth(beast) {
+    this.alive.push(beast);
+  }
```

Comment:
unnecessary static property, you can push from constructors directly to Animal.alive array